### PR TITLE
Prepare kubevirt for sonobuoy conformance testing

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

Build and release a conformance container which can be used for third parties to test ther kubevirt installation for conformance.
This initial implementation only marks one first single test  as a conformance test.

The conformance binary which will be invoked by sonobuoy will do the following:
 * by default limit test selection to tests with the tag `[Conformance]`
 * by default skip all tests with the tag `[Disruptive]`
 * tell our test binary to write the junit results to the place where sonobuoy expects it
 * tell sonobuoy when the tests are done by creating the `done` file in the expected location

To generate a sonobuoy plugin for a released conformance test suite, run

```bash
$ sonobuoy gen plugin --name kubevirt-conformance --cmd /usr/bin/conformance --image kubevirt/conformance:<tag> -f junit > kubevirt.yaml
```
    
Then luanch the suite:
    
```bash
$ sonobuoy run --plugin kubevirt.yaml
```

Here a full flow in the development environment (note the `registry:500` prefix and the `:devel` tag:

```bash
$ sonobuoy gen plugin --name kubevirt-conformance --cmd /usr/bin/conformance --image registry:5000/kubevirt/conformance:devel -f junit > conformance.yaml
sonobuoy gen plugin --name kubevirt-conformance --cmd /usr/bin/conformance --image registry:5000/kubevirt/conformance:devel -f junit > conformance.yaml
$ sonobuoy retrieve
202008201609_sonobuoy_8f8d0b0e-1d37-485a-b61d-bf7185198fbf.tar.gz
$ sonobuoy results 202008201609_sonobuoy_8f8d0b0e-1d37-485a-b61d-bf7185198fbf.tar.gz
Plugin: kubevirt-conformance
Status: passed
Total: 580
Passed: 1
Failed: 0
Skipped: 579

```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

Note, to try this out easily, I pushed the conformance binary on my docker hub account. It can be used via

```
sonobuoy gen plugin --name kubevirt-conformance --cmd /usr/bin/conformance --image docker.io/rmohr/conformance:latest -f junit > kubevirt.yaml
```

**Release note**:

```release-note
Initial support for sonobuoy for conformance testing
```

### CODE REVIEW

**Comment by rmohr** [comment]
/cc @phoracek 
/cc @EdDev 

Here a draft of what I would love to see regarding to conformance tests.

**Comment by rmohr** [comment]
> Please add some documentation under docs/. The PR description looks pretty good, it could be more or less recycled.

Done. What I don't like is that there is no testing for it. We should probably at least have a periodic job which executs the conformance tests.The user-guilde should also be updated I guess.

File: tests/utils.go [hunk 1]
```diff
@@ -1699,6 +1703,20 @@ func removeNamespaces() {
 	}
 }
 
+func detectInstallNamesapce() {
```
Comment (phoracek) [inline tests/utils.go:1706]: namespace

File: docs/conformance.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Conformance testing
+
+KubeVirt releases a `kubevirt/conformance:<release>` binary starting from
+`v.33.0`. It can be executed as a [Sonobuoy](https://sonobuoy.io/) plugin to
```
Comment (phoracek) [inline docs/conformance.md:4]: v0.33.0

**Review by phoracek — CHANGES_REQUESTED** [review]

File: tests/utils.go (hunk 1 above)
Comment (rmohr) [reply tests/utils.go]: Done

File: docs/conformance.md (hunk 2 above)
Comment (rmohr) [reply docs/conformance.md]: Done.

**Pull request merged**